R/get_settings_profile.R

```diff
@@ -64,22 +62,30 @@ get_settings_profile <- function(parameters = c("NatM_uniform_Fem_GP_1", "SR_BH_
                                  high = c(0.40, 1.0, 2),
                                  step_size = c(0.01, 0.05, 0.25),
                                  param_space = c("multiplier", "real", "relative"),
-                                 use_prior_like = c(0, 0, 0)) {
+                                 use_prior_like = lifecycle::deprecated()) 
+{
+
   if (length(parameters) != length(low) |
     length(parameters) != length(high) |
     length(parameters) != length(step_size) |
-    length(parameters) != length(param_space) |
-    length(parameters) != length(use_prior_like)) {
+    length(parameters) != length(param_space)) {
     stop("Error: input vectors do match in length.")
   }
 
+  if (lifecycle::is_present(use_prior_like)) {
+    lifecycle::deprecate_warn(
+      #when = "1.46.0",
+      what = "get_settings_profile(use_prior_like)",
+      details = "Input 'use_prior_like' no longer needed."
```

Comment:
The details argument is not needed if this is the level of detail that you are going to provide because it is clear that a deprecated argument is no longer needed. The details are for more specific things like information that the lines for both with and without a prior are now included in the figure and their is no longer an option to only have one or the other.